src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -3893,9 +3893,9 @@ components:
       pattern: ^[a-zA-Z0-9][_a-zA-Z0-9]*$
       type: string
       description: >
-        Table and column names follow this pattern. This should be used for the name of any object in the
+        Table names follow this pattern. This should be used for the name of any non-column object in the
         system. It enforces BigQuery naming rules except it disallows a leading underscore so we avoid collisions with
-        any extra columns the DR adds. For table and column names, this is shorter than what BigQuery allows.
+        any extra tables the DR adds. For table names, this is shorter than what BigQuery allows.
```

Comment:
Question for reviewers:

Since we now have a separate regex for column names, do we still want to block users from creating tables beginning with underscores?  The previous description seemed to indicate that it was a restriction needed only for columns.

Reply:
We talked about this in stand-up! We've decided to keep this existing behavior even if we're not taking advantage of it -- not allowing table names with leading underscores.